web/cypress/integration/consumer/create_and_delete_consumer.spec.js

```diff
@@ -57,6 +57,13 @@ context('Create and Delete Consumer', () => {
     cy.get(selector.description).type(data.description);
     cy.contains('Next').click();
 
+    cy.contains('Next').click();
+    cy.get(selector.notification).should(
+      'contain',
+      'Please enable at least one of the following authentication plugin:',
```

Comment:
when we do test, actually we know how many auth plugins and what they are, so we should assert the whole tip in order that we can make sure that the tip is complete